content/millennium/r4/financial/account.md

```diff
@@ -0,0 +1,147 @@
+* [Type](https://hl7.org/fhir/R4/account-definitions.html#Account.type){:target="_blank"}
+* [Subject](https://hl7.org/fhir/R4/account-definitions.html#Account.subject){:target="_blank"} 
+* [Service period](https://hl7.org/fhir/R4/account-definitions.html#Account.servicePeriod){:target="_blank"}
+* [Owner](https://hl7.org/fhir/R4/account-definitions.html#Account.owner){:target="_blank"} 
+* [Guarantor](https://hl7.org/fhir/R4/account-definitions.html#Account.guarantor){:target="_blank"} 
+* [Part of](https://hl7.org/fhir/R4/account-definitions.html#Account.partOf){:target="_blank"}
+* [Extensions including related parts, balance, and state](#extensions){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:account, :r4) %>
+
+## Extensions
+
+* [Related Parts]
+* [Balance]
+* [State]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                         | Value\[x] Type | Description
+----------------------------|----------------|-------------------------------------------------------------------------------------------
+ `related-parts`            | [`reference`]       | A reference to other related Accounts.
+ `balance`                  | [`money`]           | Represents the account balance.
+ `state`                    | [`codeableConcept`] | The status of the Account within the billing or correspondence workflow.
+
+
+## Search
+
+Search for Accounts that meet supplied query parameters:
+
+    GET /Account?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or any other required search parameter       | [`token`]     | The logical resource id associated with the resource
+ `identifier` | no                                                | [`token`]     | Aliases of the Account like Statement Number
+ `type`       | no                                                | [`token`]     | The specific type of account
+ `patient`    | This and type and identifier, or any other search param, or _id  | [`reference`] | The entity that caused the expenses
+ `-guarantor` | This and type, or any other search param, or _id  | [`reference`] | The parties responsible for balancing the account
+ `_count`     | no                                                | [`number`]    | The maximum number of results to return. Defaults to `10`.
+ 
+
+Notes:
+
+* `Account-balance` is only returned on statement and guarantor-balance types
+* `Patient` is only returned on statement and financial-account types 
```

Comment:
is "`Patient`" supposed to be "[Account.patient]" (with a corresponding link at the end of the file) ?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/586de25ae609950ca4b98aa6c40c237daa527634#diff-41181ca01179d0e3c17a1c551a4999beR68